core/storage/changes_trie/impl/changes_trie_builder_impl.cpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "storage/changes_trie/impl/changes_trie_builder_impl.hpp"
+
+#include "scale/scale.hpp"
+#include "storage/trie/impl/trie_error.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::storage::changes_trie,
+                            ChangesTrieBuilderImpl::Error,
+                            e) {
+  using E = kagome::storage::changes_trie::ChangesTrieBuilderImpl::Error;
+
+  switch (e) {
+    case E::TRIE_NOT_INITIALIZED:
+      return "No changes trie has been initialized; Supposedly startNewTrie "
+             "was not called;";
+  }
+  return "Unknown error";
+}
+
+namespace kagome::storage::changes_trie {
+
+  const common::Buffer ChangesTrieBuilderImpl::CHANGES_CONFIG_KEY{[]() {
```

Comment:
```suggestion
  static const common::Buffer ChangesTrieBuilderImpl::CHANGES_CONFIG_KEY{[]() {
```

Reply:
Wait. Why?

Reply:
Hm... Right here maybe is not, because it's singletone. But in general is good way to make common constant mebmer as static.